doc/examples/rendering/traceSetsArnold.gfr

```diff
@@ -0,0 +1,370 @@
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxOut"]["__out"], 'nodule:color', imath.Color3f( 0.240099996, 0.339399993, 0.485000014 ) )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxOut"]["__out"], 'labelPlugValueWidget:renameable', True )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxOut"]["__out"], 'nodule:type', 'GafferUI::StandardNodule' )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxOut"]["__out"], 'description', 'The processed output scene.' )
+__children["RenderSetup"]["BoxIn"]["__in"].setInput( __children["RenderSetup"]["in"] )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxIn"]["__in"], 'nodule:color', imath.Color3f( 0.240099996, 0.339399993, 0.485000014 ) )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxIn"]["__in"], 'labelPlugValueWidget:renameable', True )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxIn"]["__in"], 'nodule:type', 'GafferUI::StandardNodule' )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxIn"]["__in"], 'description', 'The input scene' )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxIn"]["__in"], 'plugValueWidget:type', '' )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxIn"]["__in"], 'noduleLayout:spacing', 2.0 )
+__children["RenderSetup"]["__uiPosition"].setValue( imath.V2f( 8.73167801, -140.832336 ) )
+__children["Backdrop"]["title"].setValue( 'Example: Trace Sets (Arnold)' )
+__children["Backdrop"]["description"].setValue( 'Arnold allows both objects and rays to be tagged with one or more trace sets. This tagging effectively compartmentalizes the scene such that affected shading calculations simply don\'t "see" objects in other trace sets.\n\nIn this example we show how to make use of Arnold\'s trace sets, by creating a mirror that only reflects some of the objects in the scene.\n\n - The spheres are tagged as being part of an Arnold trace set by a Gaffer Set node that creates a set named with a special prefix.\n\n - Rays cast from a shader are tagged by inserting a TraceSet node into the shading network before assignment.' )
+__children["Backdrop"]["__uiBound"].setValue( imath.Box2f( imath.V2f( -10, 15.4610472 ), imath.V2f( 30, 46.6711006 ) ) )
+__children["Backdrop"]["__uiPosition"].setValue( imath.V2f( -1.96337986, -25.2303677 ) )
+__children["Backdrop1"]["title"].setValue( 'Adding objects to a trace set' )
+__children["Backdrop1"]["description"].setValue( 'Gaffer manages trace sets using the same nodes as normal sets. \n\nThe only difference is that the "render:" prefix is used to denote that it should be converted to a render trace set at render time. This is currently only supported by the Arnold render plugin.\n\nHere, the Set_Spheres node adds the two spheres to a set with the name "render:spheres". This tells Gaffer to define a matching Arnold trace set called \'spheres\' at render time. ' )
```

Comment:
` \'spheres\' at render time.`
↓
` \"spheres\" at render time.`
(Or swap the string's quotation marks to singles.)

Reply:
Gah! Sorry. Fixed in https://github.com/GafferHQ/gaffer/pull/3337/commits/c67ee9a1577e9ec5d33f64b8bffc2abb58e85ab2